project/AppDependencies.scala

```diff
@@ -13,8 +13,8 @@ object AppDependencies {
     "uk.gov.hmrc"       %% "domain-play-30"                        % "9.0.0",
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"                    % mongoDbVersion,
     "uk.gov.hmrc"       %% "emailaddress-play-30"                  % "4.0.0",
-    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30"            % "8.5.0",
-    "uk.gov.hmrc"       %% "internal-auth-client-play-30"          % "1.10.0"
+    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30"            % "9.6.0",
+    "uk.gov.hmrc"       %% "internal-auth-client-play-30"          % "2.0.0"
   )
 
   val testDependencies: Seq[ModuleID] = Seq(
```

Comment:
upgrade "org.scalatest"   %% "scalatest"  to "3.2.18"
upgrade "org.scalacheck" %% "scalacheck" to "1.18.0"